src/actions/import.js

```diff
@@ -20,7 +20,7 @@ export const precheckImport = (client, url) => {
       dispatch({ type: PRECHECK_IMPORT_SUCCESS })
     } catch (e) {
       let error = 'server_error'
-      switch (e.errors && e.errors[0] && +e.errors[0].status) {
+      switch (e.status) {
```

Comment:
You are loosing the int conversion done by the `+` operator. Did you check that the switch case still work without it?

Reply:
Yes e.status return a directly that's why I made this simplification